tooling/build/package.json

```diff
@@ -11,5 +11,6 @@
   "devDependencies": {
     "@isomer/eslint-config": "*",
     "@isomer/prettier-config": "*"
-  }
+  },
+  "prettier": "@isomer/prettier-config"
```

Comment:
need run formatting again since all tooling affected hehehe

Reply:
Ah okay for some reason it didn't give me anything the first time round, fixed in d2a6419be0ac8e1ae89e1d554c3c351a50f9c8e2.